packages/video_player/video_player/lib/video_player.dart

```diff
@@ -905,16 +906,16 @@ class ClosedCaption extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
+    if (text == null || text!.isEmpty) {
```

Comment:
```suggestion
    final text = this.text;
    if (text == null || text.isEmpty) {
```

Reply:
you forgot to replace the `Text(text!)` with `Text(text)`